docs/getting-started.md

```diff
@@ -2,79 +2,661 @@
+If you've just completed the previous section, [Introduction to HarperDB Operations API](#introduction-to-harperdb-operations-api), drop the `dog` table so we can start fresh in this section.
 
-HarperDB Cloud instance provisioning typically takes 5-15 minutes. You will receive an email notification when your instance is ready.
+```sh
+# Make sure HarperDB is running (harperdb start)
+curl -X POST http://localhost:9925 \
+  -H "Content-Type: application/json" \
+  -d '{
+    "operation": "drop_table",
+    "database": "dev",
+    "table": "dog"
+  }'
+```
 
-#### Using the HarperDB Studio
+Expected output:
+```json
+{ "message": "successfully deleted table 'dev.dog'" }
+```
 
-Now that you have a HarperDB instance, if you want to use HarperDB as a standalone database, you can fully administer and interact with our database through the Studio. This section links to appropriate articles to get you started interacting with your data.
+Before continuing, shut down HarperDB using `harperdb stop`.
 
-1. [Create a database](./administration/harperdb-studio/manage-databases-browse-data.md#create-a-database)
-2. [Create a table](administration/harperdb-studio/manage-databases-browse-data.md#create-a-table)
-3. [Add a record](administration/harperdb-studio/manage-databases-browse-data.md#add-a-record)
-4. [Load CSV data](administration/harperdb-studio/manage-databases-browse-data.md#load-csv-data) (Here’s a sample CSV of the HarperDB team’s dogs)
-5. [Query data via SQL](administration/harperdb-studio/query-instance-data.md)
+### Create local environment
 
-## Administering HarperDB
+On your local machine, create a new directory for developing with HarperDB.
 
-If you are deploying and administering HarperDB, you may want to look at our [configuration documentation](deployments/configuration.md) and our administrative operations API below.
+```sh
+mkdir harperdb-getting-started
+cd harperdb-getting-started
+```
 
-### HarperDB APIs
+Generate a new Node.js project by copying the following _package.json_ file.
 
-The preferred way to interact with HarperDB for typical querying, accessing, and updating data (CRUD) operations is through the REST interface, described in the [REST documentation](developers/rest.md).
+```json
+{
+	"name": "harperdb-getting-started",
+	"private": true
+}
+```
 
-The Operations API provides extensive administrative capabilities for HarperDB, and the [Operations API documentation has usage and examples](developers/operations-api/). Generally it is recommended that you use the RESTful interface as your primary interface for performant data access, querying, and manipulation (DML) for building production applications (under heavy load), and the operations API (and SQL) for data definition (DDL) and administrative purposes.
```

Comment:
Why are we removing this? People might think they should be using the operations API for data manipulation/access (they shouldn't).